aws_lambda_powertools/utilities/data_classes/cloud_watch_alarm_event.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import annotations
+
+import json
+from enum import Enum, auto
+from typing import List, Optional
+
+from aws_lambda_powertools.utilities.data_classes.common import DictWrapper
+
+
+class CloudWatchAlarmStateValue(Enum):
+    OK = auto()
+    ALARM = auto()
+    INSUFFICIENT_DATA = auto()
+
+
+class CloudWatchAlarmState(DictWrapper):
+    @property
+    def value(self) -> CloudWatchAlarmStateValue:
```

Comment:
Instead of returning an Enum, it's better to return a Literal directly. The values are always `OK`, `ALARM` or `INSUFFICIENT_DATA` and a Literal makes this more explicit without having to create a new Enum class for it.

Reply:
Ah, coming from the TypeScript world, Python literals make me happy 😄 .